sdk/keyvault/azure-keyvault-secrets/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 4.4.0b3 (Unreleased)
 
 ### Features Added
+- Added `managed` property to SecretProperties
```

Comment:
Did you update the CHANGELOG in azure-keyvault-keys?

Reply:
I hadn't, but that's a good point! I added a note about the updated type hints, and also added another note about the multi-tenant auth changes we made in the last release of each package. That update is unrelated to this, but I needed to make a note in the changelogs so I figured I'd do it here 🙂